Sources/Xcodeproj/generate().swift

```diff
@@ -155,15 +155,17 @@ func open(_ path: AbsolutePath, body: ((String) -> Void) throws -> Void) throws
 }
 
 /// Finds directories that will be added as blue folder
-/// Excludes hidden directories, Xcode projects and reserved directories
-func findDirectoryReferences(path: AbsolutePath) throws -> [AbsolutePath] {
-    let rootDirectories = try walk(path, recursively: false)
+/// Excludes hidden directories, Xcode projects, reserved directories, and
+/// directories specified in "exclude" in the package manifest
+func findDirectoryReferences(forPackage package: Package, fileSystem: FileSystem = localFileSystem) throws -> [AbsolutePath] {
```

Comment:
Since type is clearly defined we can use `for package` instead of `forPackage package`